modules/admin_manual/pages/configuration/server/virus-scanner-support.adoc

```diff
@@ -377,3 +388,117 @@ Scanner output rules are used in daemon/socket mode.
 . A file size limit applies both to background jobs and to file uploads.
 . After installing ClamAV and the related tools, you will have two configuration files: `/etc/freshclam.conf` and `/etc/clamd.d/scan.conf`. 
 . We recommend that you enable verbose logging in both `clamd.conf` and `freshclam.conf` until you get any kinks with your ClamAV installation worked out
+
+
+== ICAP
+
+{icap-url}[ICAP] is an open standard supported by many antivirus products. With the release of the _Anti-Virus_ app 1.0.0, other virus scanners beside ClamAV can be used via ICAP if you are running it on an ownCloud Enterprise Edition. Currently the only supported virus scanners are _Kaspersky ScanEngine_ and _McAfee Antivirus_ although far more products might simply work.
+
+=== Installation
+
+- Install the Anti-Virus app 1.0.0 from the {anti-virus-app-url}[ownCloud marketplace]. Alternatively, use this occ command:
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} market:install files_antivirus
+----
+
+- Enable the app as admin in ownCloud under Settings > Apps or with the following occ command:
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} app:enable files_antivirus
+----
+
+
+=== General Configuration
+
+ICAP ist configured via occ commands. Go to your ownCloud directory, usually `/var/www/owncloud`, and enter the following occ commands:
+
+* To set the IP address of your anti-virus server:
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set files_antivirus av_host --value="172.17.0.3"
+----
+
+* Specify the port of the anti-virus server:
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set files_antivirus av_port --value="1344"
+----
+
+* Set the mode to ICAP:
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set files_antivirus av_mode --value="icap"
+----
+
+The setting 'icap' triggers a grace period of 24 hours if you don't have an Enterprise license but want to test ICAP.
```

Comment:
maybe set as a NOTE: to render differently?

Reply:
Ah, yes!